src/python/grapl_e2e_tests/tests.py

```diff
@@ -26,9 +22,3 @@ def condition() -> bool:
             return length == 4
 
         wait_for_one(WaitForCondition(condition), timeout_secs=240)
-
-
-class TestEngagementEdgeClient(TestCase):
-    def test_engagement_edge_client(self) -> None:
-        client = EngagementEdgeClient(use_docker_links=True)
-        client.get_jwt()
```

Comment:
some of this was previously in e2e tests, which frankly, only made sense because I expected to have some e2e tests that require the jwt cookie in the future